corehq/apps/reports/standard/cases/case_data.py

```diff
@@ -298,7 +297,7 @@ def _get_dd_props_by_group(domain, case_type):
         ret[prop.group_name].append(prop)
 
     uncategorized = ret.pop('', None)
-    for group, props in sorted(ret.items()):
+    for group, props in ret.items():
```

Comment:
Does this return the items in the order they were created or the `index`?  I'd expect to see a `.order_by('index')` in there.  That is, if you rearrange the properties in the DD, does this use the new ordering?  (Likewise for group ordering)

Reply:
https://github.com/dimagi/commcare-hq/pull/33098/commits/86978863149b895ef032785c6a59520c344c614c

Have added the sorting on group index and property index(Similar to [this](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/data_dictionary/views.py#L63)). Somehow previous change was also giving correct ordering for test data.

Reply:
Thanks - I was wondering about that too - a theory is that perhaps the default order is by date saved, and when editing the DD via the UI, we save all properties in sequence, meaning that the save order is the same as the index order.  Definitely safer and more explicit to add an `.order_by`, though.